include/ActionGroup.h

```diff
@@ -33,15 +32,14 @@
 /// This class provides the same functionality as QActionGroup, but in addition
 /// has the actionTriggered(int) signal.
 /// It also sets every added action's checkable property to true.
-class ActionGroup : public QActionGroup
-{
+class ActionGroup : public QActionGroup {
```

Comment:
According to the coding conventions, the opening brace should be on its own line.

Reply:
Thanks, I fixed it now.